### PR TITLE
ISAICP-6099: Featuring content should not affect last update timestamp

### PR DESCRIPTION
**Pull request by pfrenssen**

### CODE REVIEW

**Comment by pfrenssen** [comment]
I tried to do the update process in one step as was done in ISAICP-6098, but this was problematic: I was getting errors from the autoloader that the `FeaturedContentInterface` was not found when accessing the Drupal API in a normal update hook. This happens because this interface is defined in a new module, and this module is only enabled _after_ the normal update hook runs. This is a good reminder that we are not allowed to access the Drupal API in a normal update hook, only in post-update hooks.

This also indicates that we got lucky in ISAICP-6098.

File: web/modules/custom/joinup_core/joinup_core.post_update.php [hunk 1]
```diff
@@ -6,3 +6,31 @@
...
+
+    foreach ($storage->loadMultiple($entity_ids) as $entity) {
+      if ($entity instanceof FeaturedContentInterface) {
+        $entity->feature();
+        $count[$entity_type_id]++;
+      }
+    }
+  }
+
+  return 'Featured entities: ' . http_build_query($count, '', ', ');
```
Comment (claudiu-cristea) [inline web/modules/custom/joinup_core/joinup_core.post_update.php:35]: Hehe, what a usage of `http_build_query()` 👍

File: tests/features/featured_entities.feature [hunk 2]
```diff
@@ -122,16 +122,6 @@ Feature: Featuring content site-wide
...
-    # Due to the bug described in ISAICP-4352, we have to temporary add the next
-    # two lines. Manually testing or running this Behat test locally doesn't
-    # require this cache clear but it fails on the Continuous PHP bot for some
-    # obscure reasons that are very hard to be tracked.
-    # TODO: Remove these two lines in ISAICP-4849.
-    # See https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-4352.
-    # See https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-4849.
-    And the cache has been cleared
-    And I reload the page
-
```
Comment (claudiu-cristea) [inline tests/features/featured_entities.feature:134]: ❤️

**Review by claudiu-cristea — APPROVED** [review]
Nice work and so happy to see the contextual links cache issue fixed.

File: web/modules/custom/joinup_core/joinup_core.post_update.php (hunk 1 above)
Comment (pfrenssen) [reply web/modules/custom/joinup_core/joinup_core.post_update.php:35]: I found this trick on StackOverflow :grin:

**Pull request merged**